packages/opentelemetry-remote-sampler-aws-xray/package.json

```diff
@@ -0,0 +1,71 @@
+{
+  "name": "@opentelemetry/remote-sampler-aws-xray",
+  "version": "0.34.0",
+  "description": "OpenTelemetry remote sampler for AWS X-Ray",
+  "keywords": [
+    "aws",
+    "nodejs",
+    "opentelemetry"
+  ],
+  "license": "Apache-2.0",
+  "author": "OpenTelemetry Authors",
+  "bugs": {
+    "url": "https://github.com/open-telemetry/opentelemetry-js-contrib/issues"
+  },
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "files": [
+    "build/src/**/*.js",
+    "build/src/**/*.js.map",
+    "build/src/**/*.d.ts",
+    "doc",
+    "LICENSE",
+    "README.md"
+  ],
+  "repository": "open-telemetry/opentelemetry-js-contrib",
+  "scripts": {
+    "clean": "rimraf build/*",
+    "compile": "tsc -p .",
+    "lint": "eslint . --ext .ts",
+    "lint:fix": "eslint . --ext .ts --fix",
+    "precompile": "tsc --version && lerna run version:update --scope @opentelemetry/remote-sampler-aws-xray --include-dependencies",
+    "prewatch": "npm run precompile",
+    "prepare": "npm run compile",
+    "tdd": "npm run test -- --watch-extensions ts --watch",
+    "test": "nyc ts-mocha -p tsconfig.json --require '@opentelemetry/contrib-test-utils' 'test/**/*.test.ts'",
```

Comment:
These are not tests for instrumentations, so I thing `--require '@opentelemetry/contrib-test-utils'` is not needed to run them